code/src/Header.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+
+
+export const Header = () => {
+    return (
+        <section className="header"> 
+            <h1>Make somebody's day</h1>
+            <img src="./hands.png" alt="high five" />
+        </section>
+    )
```

Comment:
The header looks soo nice! But when I viewed the page I was a bit confused over where the questions were, it took me a while to get that I needed to scroll down, I clicked everywhere to se if it was something i should click. Maybe a little notice like "scroll down to answer"? :)